### DISCUSSION TITLE
Docker Actions

### DISCUSSION THREAD

**Discussion by yanniszark** [lambci/serverless-actions#5] [Q&A]
This is a great idea, congrats! One immediate thing that comes to mind is support for Docker Actions.
AFAIK, Docker can't run inside lambda. Will Docker actions be supported in this model?

**Comment by mhart** [comment]
Hey, thanks! It will support Docker Actions – but not on Lambda.

That is to say: Serverless Actions supports a number of different runner destinations. Lambda is one (and it's the fastest to launch), but there's also Google Cloud Run (which may support Docker execution in the future), AWS CodeBuild (which definitely supports Docker execution), etc, etc. You can route the job to any destination you like – even EC2 if you like.
